scripts/android-nightly.patch

```diff
@@ -20,7 +20,7 @@ index c9cd14d..f63758d 100644
  import { Colors, Header } from "react-native/Libraries/NewAppScreen";
 @@ -169,7 +169,6 @@ const App = ({ concurrentRoot }) => {
    );
- 
+
```

Comment:
This does patch still work without the whitespace? I thought they were intentional?

Reply:
not sure tbh, let me revert and just stage the line change

Reply:
Looks like it does.